screens/EventDetails/EventDetails.tsx

```diff
@@ -0,0 +1,258 @@
+            <Caption style={styles.textColor}>{event.host.handle}</Caption>
+          </View>
+        </View>
+
+        <View style={styles.iconSection}>
+          <TimeSection color={event.color} event={calenderEvent} />
+
+          <Divider style={styles.divider} />
+
+          <LocationSection
+            color={event.color}
+            coordinates={event.coordinates}
+            placemark={event.placemark}
+            bottomTabHeight={BOTTOM_TAB_HEIGHT}
+          />
+
+          <Divider style={styles.divider} />
+
+          <AttendeeSection
+            color={event.color}
+            attendeeCount={event.attendeeCount}
+          />
+
+          <Divider style={styles.divider} />
+
+          <ChatSection
+            color={event.color}
+            userAttendeeStatus={event.userAttendeeStatus}
+          />
+        </View>
+
+        {event.description && (
+          <View style={{ marginTop: 16 }}>
+            <Headline style={[styles.textColor, {marginBottom: 4 }]}>About</Headline>
+            <ExpandableText
+              props={{ style: { color: event.color, marginTop: 5 } }}
+              text={event.description}
+              linesToDisplay={3}
+            />
+          </View>
+        )}
+        <View style={{ marginTop: 16, marginBottom: BOTTOM_TAB_HEIGHT + 24 }}>
+          <Headline style={[styles.textColor, { marginBottom: 8 }]}>Location</Headline>
+          <TouchableOpacity
+            style={{
+              borderRadius: 12,
+              overflow: "hidden"
+            }}
+            onPress={openMapWithoutDirections}
+          >
+            <EventMapSnippet
+              style={{ width: "100%", height: 160 }}
+              minZoomLevel={12}
+              initialRegion={{
+                latitude: event.coordinates.latitude,
+                longitude: event.coordinates.longitude,
+                latitudeDelta: 0.1,
+                longitudeDelta: 0.1
+              }}
+              renderMarker={() => (
+                <Image
+                  source={require("../../assets/icon.png")}
+                  style={{
+                    width: MARKER_SIZE,
+                    height: MARKER_SIZE,
+                    borderWidth: 1,
+                    borderColor: "white",
+                    borderRadius: 20
+                  }}
+                />
+              )}
+              marker={{ key: event.id, location: event.coordinates }}
+            />
+          </TouchableOpacity>
+        </View>
+      </ScrollView>
+      <View style={styles.bottomTab}>
+        {isAttendingEvent(event.userAttendeeStatus)
+          ? (
+            <OutlinedButton
+              title={"Leave Event"}
+              buttonProps={{ style: styles.buttonStyle }}
+            />
+          )
+          : (
+            <PrimaryButton
+              title={"Join Now"}
+              buttonProps={{ style: styles.buttonStyle }}
+            />
+          )}
+      </View>
+      <Toast
```

Comment:
Would be good to have this in a separate component or something, we may want to reuse this for the friend screen